lib/OpenQA/WebAPI/Controller/API/V1/User.pm

```diff
@@ -0,0 +1,46 @@
+package OpenQA::WebAPI::Controller::API::V1::User;
+use Mojo::Base 'Mojolicious::Controller';
+
+=pod
```

Comment:
Instead of the POD which isn't adding much value except the doc about the method "delete" better include the license header as in other files. Or did you find another file that looks like this one that you used as a base?

Reply:
I adding the license. But I found =pod in all others files of this directory. And for instance in Command.pm we only have one method and has the =pod. Should I still remove this?

Reply:
These docs are being used by the webAPI to render the overview of API routes when you open an invalid URL so I suggest you keep it.

Reply:
fine